### PR TITLE
Added tip for local mode error

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/docs/issues/5833

### CODE REVIEW

File: source/configure/configuration-settings.rst [hunk 1]
```diff
@@ -4592,6 +4592,10 @@ This setting isn't available in the System Console and can only be set in ``conf
 | This feature's ``config.json`` setting is ``"EnableLocalMode": false`` with options ``true`` and ``false``. |
 +-------------------------------------------------------------------------------------------------------------+
 
+.. tip::
+
+  If you're seeing the error ``socket file "/var/tmp/mattermost_local.socket" doesn't exists, please check the server configuration for local mode``, this error can be resolved by setting this configuration setting to ``true``.
```
Comment (isacikgoz) [inline source/configure/configuration-settings.rst:4597]: nit: this error will come from `mmctl` so, maybe we can start the sentence with something like:
> When trying to use local mode with mmctl, having an error like ...

**Review by isacikgoz — APPROVED** [review]
It always amazes me how this kind of tips can be helpful 💪 Just one suggestion otherwise looks good.

File: source/configure/configuration-settings.rst (hunk 1 above)
Comment (cwarnermm) [inline source/configure/configuration-settings.rst:4597]:
```suggestion
  When trying to use local mode with mmctl, an error like ``socket file "/var/tmp/mattermost_local.socket" doesn't exists, please check the server configuration for local mode``, can be resolved by setting this configuration setting to ``true``.
```

File: source/manage/mmctl-command-line-tool.rst [hunk 2]
```diff
@@ -185,6 +185,10 @@ Activating local mode
 
 To use local mode, the Mattermost server first needs to `have local mode enabled <https://docs.mattermost.com/configure/configuration-settings.html#enable-local-mode>`_. When local mode is enabled, a socket is created at ``/var/tmp/mattermost_local.socket`` by default.
 
+.. tip::
+
+  If you're seeing the error ``socket file "/var/tmp/mattermost_local.socket" doesn't exists, please check the server configuration for local mode``, this error can be resolved by setting ``EnableLocalMode`` to ``true``.
```
Comment (cwarnermm) [inline source/manage/mmctl-command-line-tool.rst:190]:
```suggestion
  When trying to use local mode with mmctl, an error like ``socket file "/var/tmp/mattermost_local.socket" doesn't exists, please check the server configuration for local mode``, can be resolved by setting ``EnableLocalMode`` to ``true``.
```

File: source/configure/configuration-settings.rst (hunk 1 above)
Comment (cwarnermm) [reply source/configure/configuration-settings.rst]: Great suggestion! Thank you!

**Pull request merged**